src/main/java/net/openhft/chronicle/bytes/internal/CommonMappedBytes.java

```diff
@@ -431,7 +431,7 @@ void performClose() {
 
     @Override
     public boolean isClosed() {
-        return closeable.isClosed();
+        return closeable != null && closeable.isClosed();
```

Comment:
how can closeable be null when it is set `private final`?

Reply:
This can be called when tracing it turned on before being set.